spec/system/home_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Welcome page', type: :system do
+  it 'can see the app title' do
+    visit unauthenticated_root_path
+    expect(page).to have_content('ezPay')
+  end
+
+  it 'can see the login button' do
+    visit unauthenticated_root_path
+    expect(page).to have_content('Log in')
+  end
+
+  it 'can see the signup link' do
+    visit unauthenticated_root_path
+    expect(page).to have_content('Sign Up')
+  end
+end
```

Comment:
- [ ] Kindly write more integration tests, to cover enough cases and scenarios in your app, instead of just checking if a text exists like you are doing here. Kindly test different scenarios.

Reply:
Ok, I got it Thanks